docs/api-docs/provider-apis/tax/overview.md

```diff
@@ -34,20 +34,20 @@ To get your app ID, start creating an app in the [Developer Portal](https://devt
 | ------------------------------------- | ------------------------------ | ---------------------------- | ----------------------------------------------------------------------------------------------------- | ------------------------------------------------- |
 | App ID                                | Required                       | Integer                      | Tells us which provider config to use after the app is installed.                                     | `123456`                                          |
 | Tax provider name                     | Required                       | String                       | Displayed in the BigCommerce control panel (like in **Store Setup > Tax > Add Tax Service**).             | `Sample Tax`                                      |
-| Tax provider type                     | Required                       | Product, Sandbox             | Determines hierarchy of your provider (Production is primary and Sandbox is secondary).               | `Production`                                      |
+| Tax provider type                     | Required                       | Production, Sandbox             | Hierarchy of your provider (Production is primary and Sandbox is secondary, see [sandbox tax provider configuration](#sandbox-tax-provider-configuration) for more information.               | `Production`                                      |
 | Merchant support email                | Optional                       | Email                        | Displayed in the BigCommerce control panel in **Store Setup > Tax > Tax Provider**.                       | `support@sampletax.example.com`                   |
 | Merchant support URL                  | Optional                       | URL                          | Displayed in the BigCommerce control panel in **Store Setup > Tax > Tax Provider**.                        | `sampletax.example.com/support`                   |
 | **Coverage**                          |                                |                              |                                                                                                       |                                                   |
 | Tax provider visibility               | Required                       | Show, Hide                   | Tells us if your tax provider should be displayed to users in the BigCommerce control panel.          | `Show`                                            |
-| Supported store(s)                    | Required                       | All Stores, Private Instance | Tells us which stores your tax provider is supported on.                                              | `All Stores`                                      |
+| Supported store(s)                    | Required                       | All stores, Private instance | Tells us which stores your tax provider is supported on.                                              | `All stores`                                      |

```

Comment:
Hmm perhaps we do need to change this title, given yesterday's discussion.

This is more like "Platform Availability". This (or similar) differentiates it better from the private instance concept below and hence likely adds a bit of clarity without having to read the description.

Reply:
Updated with [latest commit](dea6ca188207b7ac2eae1415d97256124461e275). Thoughts?